ext/standard/math.c

```diff
@@ -36,7 +36,7 @@ static inline int php_intlog10abs(double value) {
 	value = fabs(value);
 
 	if (value < 1e-8 || value > 1e22) {
-		return (int)floor(log10(value));
+		return (int) floor(log10(value));
```

Comment:
Still some unrelated change here (and likewise further down).

Reply:
Does PHP have any code style change? Do you recommend something. I would be happy to learn about it as I don't want to spend time on changing formatting (I just run automatic tool for file) and/or waste your time on such minor things.

This change and next ones has been reverted.

Reply:
To the best of my knowledge there is no official code style guide. In fact the code is already inconsistently formatted (e.g. whether the `case` within a `switch` is indented or not). This is likely unavoidable due to the way PHP evolved and continues to evolve with extension-specific experts and maintainers.

What's important to me as a reviewer is that a PR does not include unrelated changes, especially not if they are purely stylistic: It distracts from the actually important changes and makes `git blame` much less useful when needing to debug older code.

Reply:
Thank you. That's reasonable.